internal/layers/layers.go

```diff
@@ -23,18 +23,25 @@ type ImageLayers struct {
 	vm                 *uvm.UtilityVM
 	containerRootInUVM string
 	layers             []string
+	// If we know the container we're unmounting the layers for is the only container in the UVM
+	// and the UVM will be killed afterwards, we shouldn't need to unmount anything.
```

Comment:
I feel like "if we know the container we're unmounting the layers for is the only container in the UVM" is confusing. Could we not just say "sandbox container"?

Reply:
This was to leave this option open to other scenarios besides a "sandbox" container. Just if this is the last container in the VM and we're going to tear down the vm afterwards. This works against me passing around a sandbox variable around however :p

Reply:
You could leave it open still by saying something like:

"In some instances we may want to avoid cleaning up an image layer, such as when tearing down a sandbox container since deletion of the UVM on the host will cleanup any left over mounts" 

Not sure if that last part is worded entirely correct, but just an idea :)

Reply:
Sure that sounds fine to me